fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -2692,6 +2693,58 @@ public LogicalPlan visitTableValuedFunction(TableValuedFunctionContext ctx) {
         });
     }
 
+    @Override
+    public LogicalPlan visitUnnestFunction(DorisParser.UnnestFunctionContext ctx) {
+        return withUnnest(ctx.unnest());
+    }
+
+    private LogicalPlan withUnnest(DorisParser.UnnestContext ctx) {
+        String defaultNestedColumnName = "unnest";
+        String defaultOrdinalityColumnName = "ordinality";
```

Comment:
maybe they should be static varaible in `Unnest`

Reply:
Done